.devcontainer/setup-user.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -e
+
+codespaces_bash="$(cat \
+<<'EOF'
+# Codespaces bash prompt theme
+__bash_prompt() {
+    local userpart='`export XIT=$? \
+        && [ ! -z "${GITHUB_USER}" ] && echo -n "\[\033[0;32m\]@${GITHUB_USER} " || echo -n "\[\033[0;32m\]\u " \
+        && echo -n "\[\033[0;36m\][IM ${IMAGEMAGICK_VERSION}]" \
+        && [ "$XIT" -ne "0" ] && echo -n "\[\033[1;31m\]➜" || echo -n "\[\033[0m\]➜"`'
+    local gitbranch='`\
+        if [ "$(git config --get codespaces-theme.hide-status 2>/dev/null)" != 1 ]; then \
+            export BRANCH=$(git symbolic-ref --short HEAD 2>/dev/null || git rev-parse --short HEAD 2>/dev/null); \
+            if [ "${BRANCH}" != "" ]; then \
+                echo -n "\[\033[0;36m\](\[\033[1;31m\]${BRANCH}" \
+                && if git ls-files --error-unmatch -m --directory --no-empty-directory -o --exclude-standard ":/*" > /dev/null 2>&1; then \
+                        echo -n " \[\033[1;33m\]✗"; \
+                fi \
+                && echo -n "\[\033[0;36m\]) "; \
+            fi; \
+        fi`'
+    local lightblue='\[\033[1;34m\]'
+    local removecolor='\[\033[0m\]'
+    PS1="${userpart} ${lightblue}\w ${gitbranch}${removecolor}\$ "
+    unset -f __bash_prompt
+}
+__bash_prompt
+
+__show_notice() {
+  local __message="
+\033[0;32mWelcome to Codespaces! You are using the pre-configured rmagick image.\033[0m
+
+\033[0;35mTests can be executed with:\033[0m bundle exec rake
+\033[0;35mCode style can be checked with:\033[0m STYLE_CHECKS=true bundle exec rubocop
+"
+  echo -e "$__message"
+
+  unset -f __show_notice
+}
+__show_notice
+EOF
+)"
+
+ echo "${codespaces_bash}" >> "/root/.bashrc"
```

Comment:
Curious, would it be possible to make the above a script rather than a string and source that instead? If `.bashrc` doesn't exist already, we could maybe copy the file or soft-link it. If it does, maybe we can append a `source /path/to/setup-user.sh`

Reply:
I copied and modified (add ImageMagick version) stuff from the codespaces own docker image that installs a lot of other dependencies that we would not need. Was also trying to get this working with a non root user and that is why I started with the script. It could probably be simplified to ` .bashrc` file and append that to the existing `/root/.bashrc` file instead. Want me to change this now or would it be okay if you did this in a follow up PR?

Reply:
Okay, no big deal. We can merge this as-is.